app/views/layouts/searches_redesign.html.haml

```diff
@@ -14,3 +15,5 @@
     = stylesheet_link_tag 'searches_redesign'
   %body
     #main-content= yield
+    = render "/shared/google_tag_manager_noscript"
+    = render "/shared/analytics"
```

Comment:
Can you move these to the `<head>`?

Reply:
Generally it's used in the body tag right?

Reply:
Moved google analytics script to `<head>` and added external tracking code at the end of `<body>` tag.